.github/workflows/release.yml

```diff
@@ -35,8 +41,8 @@ jobs:
         id: changesets
         uses: changesets/action@v1
         with:
-          publish: yarn run publish
-          version: yarn changeset version
+          publish: yarn run publish && deno publish
+          version: yarn run version:npm && yarn run version:jsr
```

Comment:
This allows changesets to first update the version in `package.json`, then copy the new version to the `deno.json`